__tests__/browser/markdown.test.js

```diff
@@ -0,0 +1,38 @@
+/* global page */
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+const port = process.env.PORT || 9966;
+
+describe('visual regression tests', () => {
+  describe('rdmd syntax', () => {
+    beforeEach(async () => {
+      await page.setViewport({ width: 800, height: 800 });
+    });
+
+    const docs = [
+      'callouts',
+      'codeBlockTests',
+      'codeBlockVarsTest',
+      'codeBlocks',
+      'embeds',
+      'features',
+      'headings',
+      'images',
+      'lists',
+      'tables',
+    ];
+
+    docs.forEach(doc => {
+      it(`renders "${doc}" without surprises`, async () => {
```

Comment:
Might want to use `it.each` instead. I've had Jest be really spotty in the past if you run tests within a loop like this.

Reply:
TIL.